automation/run-integration-tests.sh

```diff
@@ -31,7 +31,11 @@ function pyclean {
 }
 
 function docker_exec {
-    docker exec $USE_TTY -i $CONTAINER_ID /bin/bash -c "$1"
+    docker exec \
+        -e TRAVIS="$TRAVIS" \
+        -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" \
+        -e TRAVIS_BRANCH="$TRAVIS_BRANCH" \
```

Comment:
This should also pass `COVERALLS_REPO_TOKEN` to the container.

Reply:
> Please protect the COVERALLS_REPO_TOKEN.

What's the difference by putting that key to `.travis.yml` vs `..coveralls.yml`? Both no protection.

Or maybe you mean other way.

Reply:
It is possible to set environment variables at https://travis-ci.org/nmstate/nmstate/settings

Reply:
Or better see my other comment about running coveralls outside of the container.

Reply:
Good, I will use it.